### ISSUE TITLE
swarm does not expose Engine Version

### ISSUE THREAD

**Issue by paulczar** [docker/swarm#1675]
When I run `docker info` on a swarm cluster it does not provide the Engine Version.

It would be fantastic for it to provide this to me so that I know easily what Docker Engine is running on the hosts, and any apps that check the docker api for the version will not fail when trying to use Docker Engine instead.

Docker Info output showing blank engine version:

```
docker info
Containers: 4
Images: 2
Engine Version: 
Role: primary
Strategy: spread
Filters: health, port, dependency, affinity, constraint
Nodes: 1
 swarm-testing-2.novalocal: 10.230.7.96:2376
  └ Status: Healthy
  └ Containers: 4
  └ Reserved CPUs: 0 / 2
  └ Reserved Memory: 0 B / 4.057 GiB
  └ Labels: executiondriver=native-0.2, kernelversion=4.3.0-coreos, operatingsystem=CoreOS 884.0.0, storagedriver=overlay
  └ Error: (none)
  └ UpdatedAt: 2016-01-23T19:57:24Z
CPUs: 2
Total Memory: 4.057 GiB
Name: swarm-testing-2.novalocal
```
example of app that fails because it can't see this value ( and yes I know pointing a docker swarm manager at a docker swarm manager is crazy, stop judging me ):

```
$ docker run -d --name swarm-manager-2  \
    -v /etc/docker/ssl:/etc/docker/ssl \
    -p 2370:2375 swarm:latest manage \
    --tlsverify --tlscacert=/etc/docker/ssl/ca.pem \
    --tlscert=/etc/docker/ssl/cert.pem --tlskey=/etc/docker/ssl/key.pem \
    etcd://10.230.7.96:2379/swarm2

$ docker logs swarm-manager-2
time="2016-01-23T19:51:04Z" level=info msg="Listening for HTTP" addr=":2375" proto=tcp 
time="2016-01-23T19:51:04Z" level=error msg="engine 10.230.7.96:2375 is running an unsupported version of Docker Engine. Please upgrade to at least 1.6.0" 
time="2016-01-23T19:51:04Z" level=error msg="engine 10.230.7.98:2375 is running an unsupported version of Docker Engine. Please upgrade to at least 1.6.0" 
time="2016-01-23T19:51:04Z" level=error msg="engine 10.230.7.95:2375 is running an unsupported version of Docker Engine. Please upgrade to at least 1.6.0" 
```
```

**Comment by dongluochen** [comment]
@paulczar Does the error `is running an unsupported version of Docker Engine` shown in docker info after it happens?

I think it's good to show Docker Engine version in info. But in this case I think swarm manager can put the node in Pending state with the corresponding error. It can fit into #1486.

**Comment by dongluochen** [comment]
@paulczar The issue of unsupported engine version is resolved by #1683. You can see the error with `info`. If you build swarm from code, it'd be great for you to test it. 

We are working on swarm API upgrade. We may include Engine version in normal display.
